### DISCUSSION TITLE
Share translations between projects

### DISCUSSION THREAD

**Discussion by loeck** [tolgee/tolgee-platform#2436] [Q&A] [answered]
Hello,

First of all, thank you for your project. We have started using it in our technical team, and everything is running smoothly!

However, we have a need, but we're not sure if it's possible with Tolgee. To give some context:

- We have a mobile application (using Expo), which has its own project on Tolgee.
- And a web application (using Next.js), which also has its own project on Tolgee.

The need is to be able to share certain "mappings" (like API errors, for example), but we haven't found any solutions other than creating another project and duplicating bits of code to do that. Since this adds a bit too much complexity, we would prefer to just duplicate the translations, but it's not really convenient to have to change things in two places.

Any ideas or recommendations on how to handle this?

Thank you!

**Comment by JanCizmar** [comment]
Hello!

I would recommend you keeping all the strings in single project and only tag the keys with platform specific tags. The CLI tool allows you to pull only strings with specified tag.

`tolgee pull --tags mobile`

https://tolgee.io/tolgee-cli/push-pull-strings

**Reply by loeck** [reply]
Thank you for the quick response!

We suspected that, unfortunately, we use `BackendFetch` to retrieve the translations, so both mobile and web will fetch all the translations, even the ones they don't use, unless there's another solution?

**Reply by JanCizmar** [reply]
Hmm true, @stepan662. Maybe we can add a tags filter to the SDKs and the relevant backend endpoint.

**Reply by JanCizmar** [reply]
We will discuss this with the team tomorrow and let you know! 🚀

**Reply by loeck** [reply]
Perfect, thank a lot for your help 🙏

**Reply by JanCizmar** [reply, marked as answer]
Hey! We agreed that we are going to add support for this:

https://github.com/tolgee/tolgee-js/issues/3365
https://github.com/tolgee/tolgee-platform/issues/2453

**Reply by loeck** [reply]
Nice, thanks for that !
